app/Http/Controllers/Api/InventoryController1.php

```diff
@@ -0,0 +1,51 @@
+<?php namespace App\Http\Controllers;
+
+use App\Http\Requests;
+use App\Http\Controllers\Controller;
+use App\Item;
+
+use Illuminate\Support\Facades\Response;
+use Illuminate\Support\Facades\Input;
+
+class InventoryController1 extends Controller {
```

Comment:
It's generally not good to have classes named `Class1`, `Class2`. Why not rename `InventoryController1` to the more descriptive `InventoryCollectionController` and `InventoryController2` to `InventoryItemController`.